packages/flutter/lib/src/animation/animation_controller.dart

```diff
@@ -757,6 +757,14 @@ class AnimationController extends Animation<double>
     return result;
   }
 
+  /// Update the simulation without restarting the animation.
+  ///
+  /// The current simulation will be replaced with the provided [Simulation].
```

Comment:
From original PR: This needs to be clearer about behavior when the simulations (new and former) are very different. Is there a jump? Should they merge more gracefully?

Source: https://github.com/flutter/flutter/pull/100133/files#r844470742

Reply:
Also from original PR: Add more context for how and when this is used.

Source: https://github.com/flutter/flutter/pull/100133/files#r844471138